### DISCUSSION TITLE
JSON5

### DISCUSSION THREAD

**Discussion by cmdlineluser** [duckdb/duckdb#7548] [Ideas]
An interesting addition in [today's release of SQLite](https://sqlite.org/releaselog/3_42_0.html) is JSON5 read support.

https://sqlite.org/json1.html#json5

Perhaps it could be a feature worth considering for duckdb.

**Comment by lnkuiper** [comment]
DuckDB uses [yyjson](https://github.com/ibireme/yyjson) to parse JSON. We depend on the library's author to add support for JSON5 if we want to support it ourselves.
